AAI-284 issue while deleting existing chatflows and object error message

aalishanjami opened the pull request.

aalishanjami committed:
fix: confirm message on chatflow delete

aalishanjami committed:
Merge branch 'staging' into AAI-284-issue-with-deleting-existing-chatflows-and-object-error-message

aalishanjami committed:
fix: FlowListView missing prop

maxtechera reviewed (changes requested):
This PR needs to be rebased.

bradtaylorsf committed:
updated documentaton with mcp and browser extension

bradtaylorsf committed:
updated docs

bradtaylorsf committed:
removed rouge file

bradtaylorsf committed:
updated docs with succesful build

bradtaylorsf committed:
reverted back create agent button to another branch

ct3685 committed:
Merge branch 'staging' into AAI-284-issue-with-deleting-existing-chatflows-and-object-error-message

ct3685 committed:
Testing Rulesets (#212)

ct3685 committed:
Update README.md

ct3685 committed:
REMOVE ENVKEY FILE

ct3685 committed:
Create dependabot.yml

maxtechera committed:
Fix OrgId for S3 storage being undefined

ct3685 committed:
Set default chatFeedbackStatus to true and update handling for undefined status

ct3685 committed:
Set default chatFeedback status to true in chatbotConfig during save and update (#221)

ct3685 committed:
Set default chatFeedback status to true in chatbotConfig (#222)

* Set default chatFeedback status to true in chatbotConfig during save and update

* Ensure chatFeedback status is set to true by default in chatbotConfig during save and update

dependabot[bot] committed:
build(deps): bump docker/build-push-action from 5.3.0 to 6.17.0

Bumps [docker/build-push-action](https://github.com/docker/build-push-action) from 5.3.0 to 6.17.0.
- [Release notes](https://github.com/docker/build-push-action/releases)
- [Commits](https://github.com/docker/build-push-action/compare/v5.3.0...v6.17.0)

---
updated-dependencies:
- dependency-name: docker/build-push-action
  dependency-version: 6.17.0
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump pnpm/action-setup from 3 to 4

Bumps [pnpm/action-setup](https://github.com/pnpm/action-setup) from 3 to 4.
- [Release notes](https://github.com/pnpm/action-setup/releases)
- [Commits](https://github.com/pnpm/action-setup/compare/v3...v4)

---
updated-dependencies:
- dependency-name: pnpm/action-setup
  dependency-version: '4'
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump actions/checkout from 3 to 4

Bumps [actions/checkout](https://github.com/actions/checkout) from 3 to 4.
- [Release notes](https://github.com/actions/checkout/releases)
- [Changelog](https://github.com/actions/checkout/blob/main/CHANGELOG.md)
- [Commits](https://github.com/actions/checkout/compare/v3...v4)

---
updated-dependencies:
- dependency-name: actions/checkout
  dependency-version: '4'
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump docker/setup-qemu-action from 3.0.0 to 3.6.0

Bumps [docker/setup-qemu-action](https://github.com/docker/setup-qemu-action) from 3.0.0 to 3.6.0.
- [Release notes](https://github.com/docker/setup-qemu-action/releases)
- [Commits](https://github.com/docker/setup-qemu-action/compare/v3.0.0...v3.6.0)

---
updated-dependencies:
- dependency-name: docker/setup-qemu-action
  dependency-version: 3.6.0
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump docker/setup-buildx-action from 3.0.0 to 3.10.0

Bumps [docker/setup-buildx-action](https://github.com/docker/setup-buildx-action) from 3.0.0 to 3.10.0.
- [Release notes](https://github.com/docker/setup-buildx-action/releases)
- [Commits](https://github.com/docker/setup-buildx-action/compare/v3.0.0...v3.10.0)

---
updated-dependencies:
- dependency-name: docker/setup-buildx-action
  dependency-version: 3.10.0
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump connect-redis from 8.0.2 to 8.1.0

dependabot[bot] committed:
build(deps): bump @types/cors from 2.8.17 to 2.8.18

Bumps [@types/cors](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/cors) from 2.8.17 to 2.8.18.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/cors)

---
updated-dependencies:
- dependency-name: "@types/cors"
  dependency-version: 2.8.18
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

bradtaylorsf committed:
added docs build

bradtaylorsf committed:
build(deps): bump mysql2 from 3.11.3 to 3.14.1

bradtaylorsf committed:
build(deps-dev): bump vite from 5.4.14 to 6.3.5

bradtaylorsf committed:
build(deps): bump @notionhq/client from 2.2.8 to 3.1.1

bradtaylorsf committed:
build(deps): bump @oclif/core from 4.0.7 to 4.3.0

bradtaylorsf committed:
chore: update @modelcontextprotocol/sdk to 1.12.0

bradtaylorsf committed:
chore: update @langchain/google-genai to 0.2.9

bradtaylorsf committed:
chore: update @langfuse/langchain to 3.37.2

bradtaylorsf committed:
chore: update @apidevtools/json-schema-ref-parser to 12.0.2

bradtaylorsf committed:
docs: update progress - all 16 dependabot updates completed! 🎉

bradtaylorsf committed:
added docs for next time

aalishanjami committed:
Merge staging
